app/controllers/application_controller.rb

```diff
@@ -1,5 +1,13 @@
 class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
-  protect_from_forgery with: :exception
+  protect_from_forgery unless: -> { request.format.json? }
```

Comment:
this regular ApplicationController to be used by requests with Rails view templates should keep the existing protect from forgery line. This updated version with `request.format.json?` belongs in the ApiController as it's only needed when making API requests (which you already have)